terraform/README.md

```diff
@@ -9,25 +9,40 @@ Local OCI groups are used, which can be mapped to a federated group (useful when
 ## OCI Prerequisites
 
 ### Required Permissions
-You will need permission to manage the following types of resources in your OCI tenancy:
+You will need permission to manage the following types of resources in your OCI tenancy, or you may specify a specific compartment:
 * `vcns`
-* `internet-gateways`
-* `load-balancers`
+* `nat-gateways`
 * `route-tables`
 * `security-lists`
 * `subnets`
 * `instances`
+* `policies`
+* `oci_datascience_project`
+* `oci_datascience_notebook_session`
+* `oci_identity_tag_namespace`
+* `oci_identity_tag`
+
+Permissions for managing the following resource types is needed (at the tenancy level):
+* `groups`
+* `dynamic-groups`
 
```

Comment:
Permissions for DataScience service to use the following resources:
* `virtual-network-family`